Makefile

```diff
@@ -5,27 +5,61 @@ MANPAGE = genkernel.8
 # Add off-Git/generated files here that need to be shipped with releases
 EXTRA_DIST = $(MANPAGE) ChangeLog $(KCONF)
 
-default: kconfig man
-
 # First argument in the override file
 # Second argument is the base file
 BASE_KCONF = defaults/kernel-generic-config
 ARCH_KCONF = $(wildcard arch/*/arch-config)
 GENERATED_KCONF = $(subst arch-,generated-,$(ARCH_KCONF))
 KCONF = $(GENERATED_KCONF)
 
+BUILD_DIR = build
+
+FINAL_DEPS = genkernel.conf \
+	gen_cmdline.sh \
+	gen_initramfs.sh \
+	gen_determineargs.sh \
+	gen_arch.sh \
+	gen_bootloader.sh \
+	gen_compile.sh \
+	gen_configkernel.sh \
+	gen_funcs.sh \
+	gen_moddeps.sh \
+	gen_package.sh \
+	gen_worker.sh \
+	path_expander.py
+
+SOFTWARE = BCACHE_TOOLS BOOST BTRFS_PROGS BUSYBOX COREUTILS CRYPTSETUP \
+	DMRAID DROPBEAR EUDEV EXPAT E2FSPROGS FUSE GPG \
+	HWIDS ISCSI JSON_C KMOD LIBAIO LIBGCRYPT LIBGPGERROR LIBXCRYPT LVM \
+	LZO MDADM MULTIPATH_TOOLS POPT STRACE THIN_PROVISIONING_TOOLS UNIONFS_FUSE \
+	USERSPACE_RCU UTIL_LINUX XFSPROGS XZ ZLIB ZSTD
```

Comment:
one entry per line please, to ease future diffs

Reply:
addressed in commit [e87c328](https://github.com/gentoo/genkernel/commit/e87c328eac10865f4976cf2f96d3a5f86bbfe88c)

Reply:
now it's all on a single line? I expected
```
SOFTWARE = \
  BCACHE_TOOLS \
  BOOST \
  ...
```

Reply:
lmao so sorry i completely misinterpreted your comment

addressed in commit [0050718](https://github.com/gentoo/genkernel/pull/56/commits/0050718347eb34b7fe36fc3cf421abf8dfa775b9)